nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
 
+  // Reset profiling timers and counters within event_delivery_manager
+  kernel().event_delivery_manager.reset_timers_counters();
```

Comment:
I was just trying to figure out the semantics -- what gets re-run on each cycle. Since you use the counters, I'll put them where you like them, given that they're light anyhow.